src/Workspaces/VisualBasic/Portable/CodeGeneration/VisualBasicSyntaxGenerator.vb

```diff
@@ -248,6 +248,10 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.CodeGeneration
         End Function
 
         Public Overrides Function TypeExpression(typeSymbol As ITypeSymbol) As SyntaxNode
+            If typeSymbol.IsSystemVoid Then
```

Comment:
I'm not a fan of this as it means someone can't intentionally use the return type System.Void.  

When does this situation arise, and why can't the situation be solved by just using 'null' as the return type, instead of the syntax node for System.Void?